TRANSLATIONS.md

```diff
@@ -0,0 +1,32 @@
+# Translations
+
+The HTML 5.1 specification is now a stable document. We invite you to [volunteer to translate](https://www.w3.org/Consortium/Translation/) this specification, and help people around the world to reach the HTML 5.1 project.
+
+## How to Build
+
+The specification is built using [Bikeshed](https://github.com/tabatkins/bikeshed). If you would like to take advantage of the single-page.bs and build the spec when you've completed your translation, here are some suggestions for you:
+
+1. Fork the w3c/html repo
+2. Clone your fork to local: `git clone https://github.com/yourfork/html`
+3. Checkout the html5.1 branch: `git checkout html5.1`
+4. Translate the documents
+5. Install [bikeshed](https://github.com/tabatkins/bikeshed)
+6. From the HTML folder open a command prompt
+7. Run bikeshed update: `bikeshed update`
+8. Run bikeshed: `bikeshed spec`
+
+Now you have a single-page.html, you may wish to build the multipage version, please do as follows:
+
+1. Install [multipage](https://github.com/adrianba/multipage)
```

Comment:
@siusin can you please look at this? The rest looks good - although maybe TRANSLATIONS.md should be clearer that you can either translate the source, then build, or translate the published files.

Also, do the published files have the annotation and repeated CSS mess? If so it would be worth stripping those down, even manually - but you can raise an issue, tag it "editorial/when we can" and I'll do it…